clients/gtest/gemm_gtest.cpp

```diff
@@ -44,6 +44,8 @@ Representative sampling is sufficient, endless brute-force sampling is not neces
 const
 vector<vector<int>> matrix_size_range = {
                                         {-1, -1, -1, -1, 1, 1},
+                                        { 3, 33,  3,  3, 3, 3},
```

Comment:
I'm trying to validate the ldb of this line to be 3; are these column major?  If this is right, then is the {600,500, 500, 500, 600, 500} line wrong?